<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processDlg</class>
 <widget class="QDialog" name="processDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据处理进程</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_files">
             <property name="text">
              <string>文件处理队列</string>
             </property>
             <property name="buddy">
              <cstring>listWidget_files</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidget_files">
             <item>
              <property name="text">
               <string>2017年12月6日</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2017年12月5日</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_print">
             <property name="text">
              <string>打印预览</string>
             </property>
             <property name="buddy">
              <cstring>graphicsView_print</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGraphicsView" name="graphicsView_print"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_setting">
         <property name="text">
          <string>设置(&amp;S)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_printdlg">
         <property name="text">
          <string>自定义打印(&amp;C)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_quickview">
         <property name="text">
          <string>预览(&amp;V)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_next">
         <property name="text">
          <string>预览下一页(&amp;N)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_print">
         <property name="text">
          <string>打印(&amp;P)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_printall">
         <property name="text">
          <string>打印全部(&amp;A)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sendmail">
         <property name="text">
          <string>批量写入数据库并发送邮件(&amp;W)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget_files</tabstop>
  <tabstop>pushButton_setting</tabstop>
  <tabstop>pushButton_printdlg</tabstop>
  <tabstop>pushButton_quickview</tabstop>
  <tabstop>pushButton_print</tabstop>
  <tabstop>pushButton_sendmail</tabstop>
  <tabstop>graphicsView_print</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
